src/abax-client.ts

```diff
@@ -347,11 +351,11 @@ export class AbaxClient {
       .path('v1/trips')
       .query(({ input }) =>
         makeSearchParams({
-          page: input.query.page,
-          page_size: input.query.pageSize,
-          date_from: format(input.query.dateFrom, 'yyyy-MM-dd'),
-          date_to: format(input.query.dateTo, 'yyyy-MM-dd'),
-          vehicle_id: input.query.vehicleId,
+          page: input.page,
+          page_size: input.pageSize,
+          date_from: format(input.dateFrom, 'yyyy-MM-dd'),
+          date_to: format(input.dateTo, 'yyyy-MM-dd'),
+          vehicle_id: input.vehicleId,
```

Comment:
I think this would be easier to read if makeSearchParams does date formatting for you (assuming there is one universallly valid format for dates in URLSearchParams sent to Abax)